src/features/Rockets/Rockets.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import './Rockets.css';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchRockets, reserve } from './rocketSlice';
+
+function Rockets() {
+  const rockets = useSelector((state) => state.rockets.rockets);
+  const loading = useSelector((state) => state.rockets.loading);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (rockets.length === 0) dispatch(fetchRockets());
+  }, [dispatch, rockets]);
+  if (loading) {
+    return <div>Loading...</div>;
+  } return (
+    <div className="rockets">
+      {
+      rockets.map((elem) => (
+        <div key={elem.rocket_id} className="rocketTile">
+          <img src={elem.rocket_img} alt="" className="rocketimg" />
+          <div className="rocketDetails">
+            <h2 className="rocketName">{elem.rocket_name}</h2>
+            <p className="rocketDescription">
+              <span className={elem.reserved ? 'reserved' : 'notReserved'}>Reserved</span>
+              {elem.rocket_description}
+
+            </p>
+            <button onClick={() => { dispatch(reserve(elem.rocket_id)); }} type="button" className={elem.reserved ? 'reserveButton cancel' : 'reserveButton reserve'}>
+              {elem.reserved ? 'Cancel Reservation' : 'Reserve Rocket'}
+            </button>
```

Comment:
I am noticing that the image for the first rocket is not displaying appropriately and also, the color used for the button is much different from the original template (this is the correct color). Consider implementing the right color and making the image visible in order to make the UI match the template and to make it  more visually appealing

<img width="803" alt="fix" src="https://github.com/Amazinggracee/rocket-mission/assets/84930355/71924aa1-8d99-4009-84ec-a39269cf1048">